### DISCUSSION TITLE
Reddit Data cannot be processed

### DISCUSSION THREAD

**Discussion by hkim716** [pyg-team/pytorch_geometric#6773] [Q&A] [answered]
I'm trying to run ```reddit.py``` in the example codes, but I think my computer cannot process the raw data.
``` Python
import os.path as osp
import torch
from torch_geometric.datasets import Reddit

device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')

path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'Reddit')
dataset = Reddit(path)
```
I can see ``` Processing...``` but after that message, nothing happens, and nothing is in the /Processed directory.

Even in the jupyter notebook, ```The kernel appears to have died. It will restart automatically.```

Is it because the dataset is too big for my computer? (I don't think so..)
How could I get the processed data?

**Comment by rusty1s** [comment, marked as answer]
Do you see a `Downloading` log file? Otherwise, I am not totally sure. How much RAM do you have on your machine?

**Reply by hkim716** [reply]
My RAM has 16 GB. Raw data has completed downloading, but processing was not working. Could you check that again? How much size is the ```processed data```?

**Reply by hkim716** [reply]
Oh, I think 16 GB on my local machine was too small for this job. Once I dedicated more RAM, it was successfully done! Thanks Matt!
